docs/book/src/user/quick-start.md

```diff
@@ -96,7 +96,7 @@ The clusterctl CLI tool handles the lifecycle of a Cluster API management cluste
 {{#tab linux}}
 
 #### Install clusterctl binary with curl on linux
-Download the latest release; for example, to download version v0.3.0 on linux, type:
+To download version v0.3.0 on linux, type:
```

Comment:
The curl instruction we are not downloading v0.3.0, but the latest release in the 0.3 series, so IMO we should drop the example, not the first part of the sentence.

I suggest to align this to what we have on https://cluster-api.sigs.k8s.io/user/quick-start.html (
`Download the latest release; on linux, type:`)

Same below

Reply:
+1 to keep this text generic

Reply:
+1

Reply:
On a second thought, this makes sense. Updated.